packages/editor/src/components/browser/Browser.tsx

```diff
@@ -17,14 +17,16 @@ type BrowserProps = UseBrowserReturnValue & {
   types: BrowserType[];
   accept: (value: string) => void;
   location: string;
+  shortPath?: boolean;
+  onlyFile?: boolean;
```

Comment:
I think this should be an own object as it is only for the cms browser, e.g.
`cmsOptions?: {shortPath: boolean, onlyFily: boolean}`

And the only usages currently always set both on true. Are really both needed?

Reply:
At the moment, not really. I could create a flag, let's say "shortPathOnlyFile." However, I've kept them separate in case there's a need to use them independently elsewhere, such as filtering by files or outputting only the "short" path. Should I reduce it to a single flag for the moment?

Reply:
So is there a need in future for both flags?
If not I would definitely reduce this to one flag, because you have now code that will never be run...
If yes, leave it as it is, but maybe move it to a cmsOptions object

Reply:
I have now adjusted the flags:
- I put them into an own object and changed the flag from "onlyFile" to "typeFilter" where it is now possible to filter by FILE, STRING or NONE. 
- I renamed the Flag "shortPath" to "noApiCall"

Additionally I extended the search, that it also searches the content and as soon as something is entered in the search, the whole tree is opened.